integration-tests/kafka-streams/src/main/resources/application.properties

```diff
@@ -8,10 +8,11 @@ quarkus.kafka-streams.topics=streams-test-categories,streams-test-customers
 quarkus.kafka-streams.schema-registry-key=apicurio.registry.url
 quarkus.kafka-streams.schema-registry-url=http://localhost:8080
 
-quarkus.kafka-streams.security.protocol=SSL
-quarkus.kafka-streams.ssl.truststore.location=./target/classes/ks-truststore.p12
-quarkus.kafka-streams.ssl.truststore.password=Z_pkTh9xgZovK4t34cGB2o6afT4zZg0L
-quarkus.kafka-streams.ssl.truststore.type=PKCS12
```

Comment:
Is this a breaking change? It looks like it

Reply:
Or does it work in addition?

Reply:
Now it will support both.

Reply:
I can actually mix and match the properties

Reply:
We should document that somewhere - and which takes precedence when they both exist